src/firebase/CRUDServices/user.js

```diff
@@ -0,0 +1,114 @@
+export default class User {
```

Comment:
This kinda java oop/getter/setter code seems a little weird to me. I'd like more people to look at this please @harmanwho @cmutuc25 @alecherryy

Reply:
As far as I know, this isn't a thing in React js. We should probably talk about how we're handling data @harmanwho @alecherryy whenever we meet tomorrow.

Reply:
Please let me know about it. I can definitely change them to the preferred way.